index.html

```diff
@@ -6,10 +6,28 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Viva La Cumbre</title>
   <link rel="stylesheet" href="./assets/css/style.css">
```

Comment:
Adjust the path for the new/current file 

```suggestion
  <link rel="stylesheet" href="./assets/styles/style.css">
```